capstone/fitness/src/main/java/com/google/sps/servlets/GetProgressServlet.java

```diff
@@ -0,0 +1,64 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.servlets.*;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/progress")
+public class GetProgressServlet extends HttpServlet {
+
+  // Constants for datastore.
+  final static String USER_ENTITY = "user";
+  final static String PROGRESS_PROPERTY = "sessions";
+  final static String NAME_PROPERTY = "name";
+  final static String AGE_PROPERTY = "age";
+  final static String MARATHON_LENGTH_Property = "marathonLength";
+  final static String WEEKS_Property = "weeksToPrepare";
+
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    // Get the users email
```

Comment:
nit: please remember to punctuate comments, although I think we can probably remove this comment and the next one since the code seems pretty straightforward.